src/scheduler/node_info.c

```diff
@@ -4932,9 +4933,16 @@ parse_execvnode(char *execvnode, server_info *sinfo)
 		return NULL;
 	}
 
-	simplespec = parse_plus_spec_mt_safe(execvnode, &tailptr, &ret);
+	excvndup = strdup(execvnode);
```

Comment:
Use string_dup()

Reply:
I've changed it. But I'm curious, why did we decide to use malloc + strcpy inside string_dup() instead of a strdup() ?

Reply:
At the time I wrote the function, strdup() was not POSIX compliant.